src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable class-methods-use-this */
+
+import {
+  derivePathFromSeed,
```

Comment:
Are you sure getting the correct keys using this method? According BIP-32 master key should be generated with "Bitcoin seed" key and in aeternity we are using "ed25519 seed"
https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki
https://github.com/aeternity/aepp-sdk-js/blob/5df22dd297abebc0607710793a7234e6761570d4/src/utils/hd-wallet.ts#L47

Reply:
Of course not. The keys are not the same, I did compare though😊. The Bitcoin Adapter would use the bitcoinjs-lib library methods along with other implementations which is pending eg: `getBalance`